app/controllers/kudos_controller.rb

```diff
@@ -43,26 +35,23 @@ def update
     end
   end
 
-  # DELETE /kudos/1
   def destroy
     @kudo.destroy
     redirect_to kudos_url, notice: 'Kudo was successfully destroyed.'
   end
 
   def correct_employee
-    @kudo = current_employee.kudos.find_by(id: params[:id])
+    @kudo = current_employee.given_kudos.find_by(id: params[:id])
     redirect_to kudos_path, notice: 'Not Authorized To Edit This Kudo' if @kudo.nil?
   end
 
   private
 
-  # Use callbacks to share common setup or constraints between actions.
   def set_kudo
     @kudo = Kudo.find(params[:id])
   end
 
-  # Only allow a list of trusted parameters through.
   def kudo_params
-    params.require(:kudo).permit(:title, :content, :employee_id)
+    params.require(:kudo).permit(:title, :content, :employee_id, :receiver_id)
```

Comment:
I do not think, that `:employe_id` should be present in the params - in controller you are getting it from `current_user` - same thing applies to the view - I am not sure, if I as a user should have access to specify the kudo receiver and giver

Reply:
employe_id it is my giver_id
Long story short:
in Sprint 1 i did task for kudo creation before giver and receiver were require, then in acceptance criteria appeared requirement for adding giver and receiver, but i was somewhere further.
Today i want to have clean app and decided add this receiver, my giver is employee this is only semantic.
This is also the answer to your question above.